CIScripts/Test/PesterLogger/RemoteLogCollector.ps1

```diff
@@ -12,6 +12,12 @@ function New-LogSource {
     }
 }
 
+function New-ContrailLogsSource {
+    Param([Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions)
+
+    New-LogSource -Sessions $Sessions -Path "C:/ProgramData/Contrail/var/log/contrail/*.log"
```

Comment:
Hm.. I think I disagree with this diff. We shouldn't put anything related to log paths in a helper module. We want it to be pure.

If anything, this function should be in `Testbed.ps1` or some other file in `Testenv/` directory.

Reply:
@jacek-izykowski what do you think